python/pyspark/rdd.py

```diff
@@ -2169,9 +2169,7 @@ def name(self):
         """
         Return the name of this RDD.
         """
-        n = self._jrdd.name()
-        if n:
-            return n
+        return self._jrdd.name()
```

Comment:
Maybe, we should avoid unrelated change although I understand this is semantically clear.